mne/commands/tests/test_commands.py

```diff
@@ -276,6 +277,36 @@ def test_flash_bem(tmpdir):
         assert len(tris) == 5120
 
 
+@pytest.mark.slowtest
+@sample.requires_sample_data
```

Comment:
Last thing -- this shouldn't require the `sample` dataset but instead the `testing` dataset. By default Travis and AppVeyor do not get the `sample` dataset, so it's better to use `testing` if possible. And I think the tests you do here should be doable with `testing`.